### PR TITLE
I found some acronyms.

### PR DESCRIPTION
**Pull request by samcunliffe**
Part of https://github.com/inform-us/INFORMus/issues/1018

### CODE REVIEW

**Comment by samcunliffe** [comment]
Also CCU is used without [introduction here](https://github.com/inform-us/INFORMus/files/14039311/InformUs_AnnualReport_2022.without.appendix.section.V11.pdf). I wasn't sure if critical care unit (I'm assuming) or coronary care unit (also used...??).

File: Glossary.md [hunk 1]
```diff
@@ -295,6 +299,10 @@
 	Strategic Integration Platform acts as a central controller for all electronic messages sent within the 
     Trust. It routes messages from various sources ([POC](#poc), [HSL](#hsl), Imaging) to be recorded in the 
     appropriate databases.
+
+- <a id="star">**SPC**</a>
+
+	Statistical Process Chart (or Control).
```
Comment (SJaffa) [inline Glossary.md:305]:
```suggestion
	Statistical Process Chart Control.

```
I think they are referred to as SPC charts so the C is control. This matches how they are described in out [InformUs rules documents](https://github.com/inform-us/requirements_specifications/blob/main/rules/SPCchartbuild.md)

Comment (samcunliffe) [reply Glossary.md:305]: I now think you're right. I was just going on this...

Screenshot 2024-01-25 at 15 21 24

**Review by SJaffa — APPROVED** [review]

**Comment by samcunliffe** [comment]
I actually can't merge this because I don't have "write".
Screenshot 2024-02-05 at 11 19 49

p.s. I don't think I need "write" here for anything other than merging this ... PoLP.

**Pull request merged**